src/libs/Environment/Environment.js

```diff
@@ -31,8 +37,21 @@ function getEnvironmentURL() {
     });
 }
 
+/**
+ * Get the corresponding oldDot URL based on the environment we are in
+ *
+ * @returns {Promise}
+ */
+function getOldDotEnvironmentURL() {
+    return new Promise((resolve) => {
+        getEnvironment()
+            .then(environment => resolve(OLDDOT_ENVIRONMENT_URLS[environment]));
+    });
```

Comment:
Wrapping with `new Promise((resolve => )` is unnecessary - `getEnvironment` already returns a promise you can just do this:

```js
return getEnvironment()
  .then(environment => OLDDOT_ENVIRONMENT_URLS[environment]);
```